### ISSUE TITLE
[modal] Confirm retirement modal UI tweak

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#1963]
### To replicate:

- Login and go to your portfolio page
- Select a project/asset to retire and click `RETIRE` button
- On the retire form ([for example](https://www.carbonmark.com/portfolio/0x055551c2c1d739740461ce9192fccdb8e62ad89d/retire)) select amount and click `RETIRE CARBON` button
- In the Confirm Retirement modal, on both the APPROVE and SUBMIT tabs, note that the Submit amount has a USDC logo
<img width="544" alt="Screenshot 2023-12-04 at 9 47 48 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/c78ad6b6-d064-4ccd-9ba9-f1600a79b4b0">

### The issue:
- Retiring an asset you already own does not require USDC or any form of payment
- What the user is confirming in this modal is actually the quantity (tonnes of carbon to retire)
- I believe this is happening because we are reusing the component when retiring from a pool listings ([for example](https://www.carbonmark.com/projects/VCS-1547-2020/retire/pools/ubo)) which does require USDC payment. In this case the Confirm amount is the Total cost showing the USDC symbol makes sense
<img width="472" alt="Screenshot 2023-12-04 at 9 53 08 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/217fff61-9583-49a6-8751-a8cefb2b8f2a">

### Design question

- [ ] @theprofessora Should Confirm amount in these modals be the amount of carbon or the in the case of retire from a pool was they intent that the Confirm amount was the total cost?
- [ ] @theprofessora depending on answer to above, should we standardize these or do we need to support one modal that shows cost and one modal that shows quantity?

**Comment by theprofessora** [comment]
Yes, the confirm should be the amount to retire. I think we should have one modal that shows all necessary fields.

So if there's a payment required, we should have 3 fields: contract address, confirm price, confirm quantity.

If there's no payment required, we should have 2 fields: contract address, confirm quantity.

I'll update the designs in figma
